modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java

```diff
@@ -640,10 +641,12 @@ protected Response addMediaPackageElement(HttpServletRequest request, MediaPacka
           @RestParameter(description = "URL of series DublinCore Catalog", isRequired = false, name = "seriesDCCatalogUri", type = RestParameter.Type.STRING),
           @RestParameter(description = "Series DublinCore Catalog", isRequired = false, name = "seriesDCCatalog", type = RestParameter.Type.STRING),
           @RestParameter(description = "Access control list in XACML or JSON form", isRequired = false, name = "acl", type = RestParameter.Type.STRING),
+          @RestParameter(description = "Flavor of the next media file", isRequired = false, name = "flavor", type = RestParameter.Type.STRING),
```

Comment:
Why are we duplicating the "flavor" parameter here? It's already specified as required earlier? Looks good otherwise.

Reply:
You are right → fixed.